tests/unit/helpers/dates.test.js

```diff
@@ -145,7 +145,7 @@ describe('dayDiff', () => {
 
     const result = DatesHelper.dayDiff(date1, date2);
 
-    expect(result).toBe(0);
+    expect(result).toBe(1);
   });
 
   it('should return number of days between two dates', () => {
```

Comment:
Est-ce que tu pourrais renommer les `it` de ces tests pour preciser un peu ce que fait chaque test ?